neat_ml/tests/test_workflow.py

```diff
@@ -0,0 +1,179 @@
+import logging
+from pathlib import Path
+from typing import Any
+import pytest
+
+import neat_ml.workflow.lib_workflow as wf
+
+
+def test_get_path_structure_builds_expected_paths(tmp_path: Path) -> None:
```

Comment:
Here and elsewhere -- the `-> None` isn't really need here, I don't think, especially in test functions that would obviously not return something.